### PR TITLE
Update amp-ad-exit docs

### PR DESCRIPTION
**Pull request by clawr**
to reflect changes in https://github.com/ampproject/amphtml/pull/11462

### CODE REVIEW

File: extensions/amp-ad-exit/amp-ad-exit.md [hunk 1]
```diff
@@ -273,8 +276,7 @@ Example:
         },
         "_3pAnalytics": {
           "defaultValue": "no_response",
-          "vendorAnalyticsSource": "VendorXYZ",
-          "vendorAnalyticsResponseKey": "findings"
+          "iframeTransportSignal": "IFRAME_TRANSPORT_SIGNAL(example-3p-vendor,collected-data)"
```
Comment (jonkeller) [inline extensions/amp-ad-exit/amp-ad-exit.md]: LGTM. You might add a note that (at least for now) adding a space after this comma is not allowed. (See https://github.com/ampproject/amphtml/pull/11461#discussion_r142504598 )

Comment (clawr) [reply extensions/amp-ad-exit/amp-ad-exit.md]: Thanks, done

**Comment by lannka** [comment]
@clawr shall we merge this?

File: extensions/amp-ad-exit/amp-ad-exit.md (hunk 1 above)
Comment (jonkeller) [reply extensions/amp-ad-exit/amp-ad-exit.md]: (Note that the comma thing is no longer true...)

Comment (lannka) [reply extensions/amp-ad-exit/amp-ad-exit.md]: @calebcordry is working on nested function calls in URL replacement. we might again restrict the space use here.

Comment (calebcordry) [reply extensions/amp-ad-exit/amp-ad-exit.md]: I am not 100% yet. There might be a way to preserve spacing after comma only. I am still thinking it through.

**Pull request merged**